CRM/Contact/BAO/Group.php

```diff
@@ -1130,12 +1130,12 @@ private static function buildGroupHierarchy(&$hierarchy, $group, $tree, $titleOn
    */
   public static function getGroupCount(&$params) {
     $whereClause = self::whereClause($params, FALSE);
-    $query = "SELECT COUNT(*) FROM civicrm_group groups";
+    $query = "SELECT COUNT(*) FROM civicrm_group `groups`";
 
     if (!empty($params['created_by'])) {
       $query .= "
 INNER JOIN civicrm_contact createdBy
```

Comment:
@seamuslee001 did you missed this alias ? 
```patch
- INNER JOIN civicrm_contact createdBy
+ INNER JOIN civicrm_contact `createdBy`
- ON createdBy.id = groups.created_id";
+ ON `createdBy`.id = `groups`.created_id";
```

Reply:
createdBy as far as i know is not a reserved word in MySQL therefore it doesn't need a ` around it

Reply:
oh ok